src/api.rs

```diff
@@ -240,6 +242,26 @@ impl RapiClient for RapiReqwestClient {
             form = form.text("name", name)
         }
 
+        if let Some(env_args) = env_args {
+            for env_arg in env_args {
+                let key_value: Vec<&str> = env_arg.splitn(2, '=').collect();
```

Comment:
You can optimize this by removing collect() and just using for_each since you're not really using the resulting vector